### DISCUSSION TITLE
Do we still need `FUEL_NETWORKL_URL`? 🤔

### DISCUSSION THREAD

**Discussion by petertonysmith94** [FuelLabs/fuels-ts#2510] [Ideas]
[Full context](https://github.com/FuelLabs/fuels-ts/pull/2491#discussion_r1634647256)

> @Torres-ssf `FUEL_NETWORK_URL` constant is mainly ( if not only ) used within our tests.

> @danielbate The one caveat is that when copying from snippets, FUEL_NETWORK_URL used a ternary that could evaluate but live and local networks, so it could be extracted from snippets. Whereas someone now will need to change the constant dependent on if they are working on testnet or locally.

> @arboleya I wonder if such a constant should be static and avoid the ternary.

What do we want to do about `FUEL_NETWORK_URL`?

**Comment by arboleya** [comment]
TL;DR of my proposal:
 - `DEVNET_NETWORK_URL`
 - `TESTNET_NETWORK_URL`
 - `NETWORK_URL` — upcoming `mainnet`

I also don't think we need a `LOCAL_NETWORK_URL` because it's user-defined and, when combined with the `fuels` CLI and `launch{Test}Node,` the port is dynamic.

**Comment by petertonysmith94** [comment]
Will `FUEL_NETWORK_URL` also be made redundant by the `launchTestNode` integration?

How do we want to display the URL in the documentation? 🤔

**Reply by arboleya** [reply]
Yes, if you use `launchTestNode,` you should not need any constant.

I think could ditch `FUEL_NETWORK_URL` and keep only:

- `DEVNET_NETWORK_URL`
- `TESTNET_NETWORK_URL`
- `NETWORK_URL` — upcoming `mainnet`

**Reply by petertonysmith94** [reply]
> Yes, if you use `launchTestNode,` you should not need any constant.
> 
> I think could ditch `FUEL_NETWORK_URL` and keep only:
> 
> * `DEVNET_NETWORK_URL`
> * `TESTNET_NETWORK_URL`
> * `NETWORK_URL` — upcoming `mainnet`

I tend to agree - created https://github.com/FuelLabs/fuels-ts/issues/2644 for this

**Comment by petertonysmith94** [comment]
Discussion concluded
